src/styles/normalize.scss

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
```

Comment:
`margin: 0;` it's forbidden to use anything except box-sizing: border-box; in `*`